index.js

```diff
@@ -43,11 +43,8 @@ function byPrecision (a, b) {
 }
 
 function override (script) {
-  var pathToPinoDebug = require.resolve('./debug')
+  var pathToPinoDebug = require.resolve('./debug').replace(/\\/g, '/')
```

Comment:
I'm confused about why this wouldn't work on Windows. Doesn't Node resolve such paths internally to the correct path structure for the current platform?

Regardless, I think this change would be better as:

```js
var pathToPinoDebug = path.resolve(path.join(__dirname, 'debug'))
```

Reply:
Hi @jsumners ,

I need put this additional replace to change windows backslashes to forward slashes, thank to this operation when this path is inserted in string (for `head` veriable)

```javascript
  var head = `(function(exports, require, module, __filename, __dirname) {
      require = (function (req) {
        var Object = ({}).constructor
        return Object.setPrototypeOf(function pinoDebugWrappedRequire(s) {        
          var dirname = __dirname.slice(-22)
          var pathToPinoDebug = '${pathToPinoDebug}'
```
code works correctly, but without this when I just put path directly from `require.resolve('./debug')` or `path.resolve(path.join(__dirname, 'debug'))` into `head` variable I receive this error

```
  not ok Error: Cannot find module 'C:ProjektyLearnpino-debugdebug'
    at:
      line: 489
      column: 15
      file: module.js
      function: Function.Module._resolveFilename
    stack: |
      pinoDebugWrappedRequire (node_modules/debug/src/node.js:1:444)
      Object.<anonymous> (node_modules/debug/src/node.js:14:28)
      pinoDebugWrappedRequire (node_modules/debug/src/index.js:1:689)
      Object.<anonymous> (node_modules/debug/src/index.js:9:20)
    code: MODULE_NOT_FOUND
    test: keeps line numbers consistent
    message: 'Error: Cannot find module ''C:ProjektyLearnpino-debugdebug'''

```
As you can see backslashes was removed from path `C:ProjektyLearnpino-debugdebug` and node.je report that he cannot found that module.